RUFAS/biophysical/feed_storage/feed_manager.py

```diff
@@ -311,7 +311,11 @@ def get_total_inventory(self, inventory_date: date, weather: Weather, time: Rufa
 
         available_feed_rufas_ids = [feed.rufas_id for feed in self._available_feeds]
 
-        available_feed_totals = self._query_available_feed_totals(available_feed_rufas_ids, projected_crops)
+        projected_shrinkage = self.purchased_feed_storage.project_shrinkage(days_in_the_future, self._available_feeds)
```

Comment:
```suggestion
        projected_shrunk_purchased_feeds = self.purchased_feed_storage.project_shrinkage(days_in_the_future, self._available_feeds)
```
Doesn't have to be this but I think it should be clear this is the list of purchased feeds that have had shrink projected out for them.